manifests/c/CalcProgrammer1/OpenRGB/0.9/CalcProgrammer1.OpenRGB.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: CalcProgrammer1.OpenRGB
+PackageVersion: "0.9"
+InstallerLocale: en-US
+InstallerType: zip
+ReleaseDate: 2022-07-09
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: './OpenRGB Windows 32-bit/OpenRGB.exe'
+    PortableCommandAlias: openrgb
+  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_32_b5f46e3.zip
+  InstallerSha256: 983CC04BC14FCB824E8075C23BD9A39642B632C84784510FF81DD8FDD84331B4
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: './OpenRGB Windows 64-bit/OpenRGB.exe'
+    PortableCommandAlias: openrgb
+  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_64_b5f46e3.zip
+  InstallerSha256: 4A42DF973BF9E0694268993478F03A71DAFBF2DDBCB1512835B4BBABDC6DC6DE

```

Comment:
```suggestion
Installers:
- Architecture: x86
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: './OpenRGB Windows 32-bit/OpenRGB.exe'
    PortableCommandAlias: openrgb
  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_32_b5f46e3.zip
  InstallerSha256: 983CC04BC14FCB824E8075C23BD9A39642B632C84784510FF81DD8FDD84331B4
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: x64
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: './OpenRGB Windows 64-bit/OpenRGB.exe'
    PortableCommandAlias: openrgb
  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_64_b5f46e3.zip
  InstallerSha256: 4A42DF973BF9E0694268993478F03A71DAFBF2DDBCB1512835B4BBABDC6DC6DE
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

Reply:
Hi @Macelykun,

I believe that installing both frameworks might be causing the error above.